curriculum/challenges/english/03-front-end-libraries/redux/never-mutate-state.english.md

```diff
@@ -25,7 +25,7 @@ tests:
   - text: The Redux store should exist and initialize with a state equal to the <code>todos</code> array in the code editor.
     testString: assert((function() { const todos = [ 'Go to the store', 'Clean the house', 'Cook dinner', 'Learn to code' ]; const initialState = store.getState(); return Array.isArray(initialState) && initialState.join(',') === todos.join(','); })(), 'The Redux store should exist and initialize with a state equal to the <code>todos</code> array in the code editor.');
   - text: <code>addToDo</code> and <code>immutableReducer</code> both should be functions.
-    testString: assert(typeof addToDo === 'function' && typeof immutableReducer === 'function', '<code>addToDo</code> and <code>immutableReducer</code> both should be functions.');
+    testString: assert(typeof addToDo === 'function' && typeof immutableReducer === 'function');
   - text: Dispatching an action of type <code>ADD_TO_DO</code> on the Redux store should add a <code>todo</code> item and should NOT mutate state.
     testString: assert((function() { const initialState = store.getState(); const isFrozen = DeepFreeze(initialState); store.dispatch(addToDo('__TEST__TO__DO__')); const finalState = store.getState(); const expectedState = [ 'Go to the store', 'Clean the house', 'Cook dinner', 'Learn to code', '__TEST__TO__DO__' ]; return( isFrozen && DeepEqual(finalState, expectedState)); })(), 'Dispatching an action of type <code>ADD_TO_DO</code> on the Redux store should add a <code>todo</code> item and should NOT mutate state.');
```

Comment:
Can we come up with something to remove these ones?

Reply:
@thecodingaviator I have now made a small tweak to a regular expression to capture ones like this that were missed in my previous commits.  I will go through the other PRs and run my script again to see if it captures more like these two you pointed out here.

Thanks for finding these.